packages/manager/apps/container/src/container/nav-reshuffle/sidebar/style.module.scss

```diff
@@ -8,9 +8,14 @@
   height: 100%;
   display: flex;
   flex-direction: column;
+  justify-content: flex-end;
   min-width: 20rem;
   color: white;
 
+  &_menu {
+    // overflow: scroll;
```

Comment:
Should not be commented, if it is, sidebar won't work on smaller screen sizes when there are a lot of elements and Assistance is Open

We can see the issue in the staging

Reply:
It breaks the display in some case. We will discuss about it and provide a solution.